README.md

```diff
@@ -11,32 +11,30 @@ third party services nor special IDE plugins.
 [![PyPI](https://img.shields.io/pypi/l/django-sass-processor.svg)]()
 [![Twitter Follow](https://img.shields.io/twitter/follow/shields_io.svg?style=social&label=Follow&maxAge=2592000)](https://twitter.com/jacobrief)
 
-**django-sass-processor** converts ``*.scss`` or ``*.sass`` files into ``*.css`` while rendering
+**django-sass-processor** converts `*.scss` or `*.sass` files into `*.css` while rendering
```

Comment:
Note: markdown changes were auto-made by my editor's Prettier plugin when I've added the changelog entry. Can revert if you wish.